crowdkit/aggregation/pairwise/bradley_terry.py

```diff
@@ -14,43 +14,45 @@
 
 @attr.s
 class BradleyTerry(BasePairwiseAggregator):
-    r"""Bradley-Terry model for pairwise comparisons.
-
-    The model implements the classic algorithm for aggregating pairwise comparisons.
-    The algorithm constructs an items' ranking based on pairwise comparisons. Given
-    a pair of two items $i$ and $j$, the probability of $i$ to be ranked higher is,
-    according to the Bradley-Terry's probabilistic model,
+    r"""The **Bradley-Terry model for paired comparisons** implements the classic algorithm
+    for aggregating paired comparisons. The algorithm constructs the ranking of items based on paired comparisons.
+    Given a pair of two items $i$ and $j$, the probability that $i$ is ranked higher than $j$,
+    according to the probabilistic Bradley-Terry model, is
     $$
-    P(i > j) = \frac{p_i}{p_i + p_j}.
+    P(i > j) = \frac{p_i}{p_i + p_j},
     $$
-    Here $\boldsymbol{p}$ is a vector of positive real-valued parameters that the algorithm optimizes. These
-    optimization process maximizes the log-likelihood of observed comparisons outcomes by the MM-algorithm:
+    where $\boldsymbol{p}$ is a vector of the positive real-valued parameters that the algorithm optimizes. These
+    optimization process maximizes the log-likelihood of the outcomes of the observed comparisons using the MM algorithm:
     $$
     L(\boldsymbol{p}) = \sum_{i=1}^n\sum_{j=1}^n[w_{ij}\ln p_i - w_{ij}\ln (p_i + p_j)],
     $$
-    where $w_{ij}$ denotes the number of comparisons of $i$ and $j$ "won" by $i$.
+    where $w_{ij}$ denotes the number of times individual $i$ has beaten individual $j$ and we assume $w_{ij} = 0$ by convention.
```

Comment:
Not an "individual", I suppose it's better to say "object" here. Also, don't understand why does w_ij equals zero.